README.md

```diff
@@ -1 +1,78 @@
-# psc-verification-api
\ No newline at end of file
+# psc-verification-api
+
+The People with Significant Control (PSC) Verification API is a Spring Boot REST API which forms part of the Identification Verification (IDV) service and is responsible for handling and processing PSC Individual and Relevant Legal Entity (RLE) officer verification requests.
+
```

Comment:
Not sure, but should the term be Relevant Officer? -  "Relevant Legal Entity (RLE) Relevant officer"